app/views/claims/pages/index.html.erb

```diff
@@ -1,29 +1,56 @@
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">It works! 🎉</h1>
+<%= navigation_bar %>
+<% if @user&.schools&.first&.gias_school %>
+    <div class="govuk-grid-row">
+    <br><br>
+      <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l"><%= t("organisation_details") %></h1>
+      <%= govuk_summary_list do |summary_list|
+            summary_list.with_row do |row|
+              row.with_key(text: t("organisation_name"))
+              row.with_value(text: @user&.schools&.first&.gias_school&.name)
```

Comment:
To respect the Law of Demeter , we should delegate the attributes we want to access in the `gias_school` model - I've just checked, we already do this.

To also avoid `nil` errors, we can add the following:

```ruby
class School < ApplicationRecord
  belongs_to :gias_school, foreign_key: :urn, primary_key: :urn

  delegate :name, to: :gias_school, allow_nil: true # Add allow_nil
end
```

This will allow you to do `school.name` and it will return `nil` when the `gias_school` association is missing.

As for `@user`, shouldn't that always be the `current_user` in this case?

Also, if you're using the `/schools/:school_id` path, you should be able to do:

```ruby
class Claims::SchoolsController < ApplicationController
  before_action :set_school

  private

  def set_school
    current_user.schools.find(params[:id])
  end
end
```

If the User does not belong to the School, the `#find` call will raise a not found error.

Reply:
> This will allow you to do school.name and it will return nil when the gias_school association is missing.

Just to chime in here – I think we should avoid using `allow_nil: true` here until we have a clear need for it.

A School should _always_ have an associated GiasSchool. And if it doesn't, something has gone wrong. Given that this should only happen in exceptional circumstances, I'd rather we end up finding the exception in Sentry rather than it silently failing and users are presented with schools that have no name.

So to be clear: I think it should be entirely safe to assume we can always call `school.name` without having to worry about missing GiasSchool associations.

Reply:
The relationship between user and school didn't exist when i started the ticket. The path is NOT /schools/:school_id. It is under /. In the ticket they were no mention or indication that's what we would want. If you take a look at the prototype. When you sign in for school you only have access to one school. 
Most of this changes are necessary but probably better for a another ticket . I m going to use fake data for now and for clarity sake.

Reply:
Please move the code into the `/schools/:school_id` path. The data can be entirely fake.